frb_codegen/src/config.rs

```diff
@@ -222,7 +215,7 @@ pub fn parse(raw: RawOpts) -> Vec<Opts> {
                 manifest_path: manifest_paths[i].clone(),
                 dart_root: dart_roots[i].clone(),
                 build_runner, //same for all rust api blocks
-                exclude_sync_execution_mode_utility: exclude_sync_execution_mode_utilities[i],
+                block_index: i + 1,
```

Comment:
If count from zero +1 is no longer needed

Reply:
Fixed in [cc7cfbc](https://github.com/fzyzcjy/flutter_rust_bridge/pull/527/commits/cc7cfbc4f1a0b944bff5d25383e8f87edc1a914f).